app/controllers/selections_controller.rb

```diff
@@ -1,18 +1,37 @@
+include SelectionsHelper
+
 class SelectionsController < ApplicationController
 
-  def add
-    @user = User.find(session[:user_id])
-    @station = Station.all
-    # figure this out
-    @selection = Selection.new(params:user_id, station_id)
+  def index
+    @selections = current_user.stations    #provides an array of station objects
+    @forecasts = []
+    get_forecast(@selections)
```

Comment:
This should be 
```
@forecasts = get_forecast(@selections)
```

Reply:
Makes sense.

Reply:
but now I'm getting a bunch of 'no implicit conversion of Symbol into Integer' errors. working on it...